lib/ConstantFraction.js

```diff
@@ -68,12 +68,14 @@ ConstantFraction.simplifyFraction = function(fraction) {
     MathChangeTypes.SIMPLIFY_FRACTION, oldFraction, newFraction);
 };
 
-// If `node` is a sum of constant fractions, either finds the LCD and sets
-// up all the denominators to equal the LCD, or adds the fractions together
-// if they all have the same denominator.
-// e.g. 2/3 + 4/6 --> (2*2)/(3*2) + 4/6
-// e.g. 2/5 + 4/5 --> (2+4)/5
-// returns a NodeStatus object.
+// Adds constant fractions -- can start from either step 1 or 2
+// 1A. Find the LCD if denominators are different and multiplies to make
+//     denominators equal, e.g. 2/3 + 4/6 --> (2*2)/(3*2) + 4/6
+// 1B. Multiplies out to make constant fractions again
+//     e.g. (2*2)/(3*2) + 4/6 -> 4/6 + 4/6
+// 2A. Combines numerators, e.g. 4/6 + 4/6 ->  e.g. 2/5 + 4/5 --> (2+4)/5
+// 2B. Adds numerators together, e.g. (2+4)/5 -> 6/5
```

Comment:
I think because of the changing around of oldNode and newNode and subSteps, calling a function and then updating the variables would be just as much code and a bit less clear

I made this area a bit cleaner by calling functions e.g. `  let newNode = combineNumeratorsAboveCommonDenominator(oldNode.args);` so I'm okay with how it is right now